projects/addon-mobile/components/sheet/directives/sheet-stop/sheet-stop.directive.ts

```diff
@@ -31,15 +29,15 @@ export class TuiSheetStopDirective {
                 map(([above, dragged]) => !above && !dragged),
                 filter(Boolean),
                 throttleTime(100),
-                takeUntil(destroy$),
+                takeUntilDestroyed(destroy$),
```

Comment:
`destroy$` is unnecessary

Reply:
We have nested scope

<img width="581" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/595f935d-27ef-431f-a5bb-036f8dd3c9a8">